src/models/campaigns.model.js

```diff
@@ -38,6 +40,7 @@ function createModel(app) {
       url: { type: String },
       commitTime: { type: Number },
       communityUrl: { type: String },
+      token: { type: Token }
```

Comment:
why do campaigns & dacs need a token? Shouldn't this only be on the milestone & donations models?